File: Cpp/8 kyu Even or Odd/evenOrOdd.cpp
/*
Create a function that takes an integer as an argument and returns
"Even" for even numbers or "Odd" for odd numbers.

Examples:
evenOrOdd(2) -> "Even"
evenOrOdd(7) -> "Odd"
evenOrOdd(-42) -> "Even"
*/

// Solution

std::string even_or_odd(int number) {
  return number % 2 == 0 ? "Even" : "Odd";
}

File: Php/8 kyu Even or Odd/even_or_odd.php
/*
Create a function that takes an integer as an argument and returns
"Even" for even numbers or "Odd" for odd numbers.

Examples:
even_or_odd(2) -> "Even"
even_or_odd(7) -> "Odd"
even_or_odd(-42) -> "Even"
*/

// Solution

function even_or_odd(int $number): string {
    return $number % 2 === 0 ? 'Even' : 'Odd';
}
